modules/tinymce/src/themes/silver/test/ts/webdriver/tooltip/HtmlPanelTooltipTest.ts

```diff
@@ -1,19 +1,20 @@
 import { RealKeys, RealMouse } from '@ephox/agar';
 import { context, describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
 import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
 
 import Editor from 'tinymce/core/api/Editor';
 
 import * as TooltipUtils from '../../module/TooltipUtils';
 
 describe('browser.tinymce.themes.silver.editor.HtmlPanelTooltipTest', () => {
-
   const pOpenDialogAndWaitForLoad = async (editor: Editor, buttonSelector: string) => {
     TinyUiActions.clickOnToolbar(editor, buttonSelector);
     await TinyUiActions.pWaitForDialog(editor);
   };
 
-  context('Tooltip if data-mce-tooltip is in htmlpanel', () => {
+  const contextFn = PlatformDetection.detect().browser.isSafari() ? context.skip : context;
```

Comment:
We probably need a comment or link to a Jira here to explain why this is needed

Reply:
I've created a ticket TINY-10995 to keep track of this 👍